### PR TITLE
Remove template editor

### PR DESCRIPTION
**Pull request by Justarandomamerican**
The user who requested it went inactive ages ago, and it is not used for testing, nor has any reasonable future use. I suggest removing it.

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -358,16 +358,14 @@
 $wgGroupPermissions['chatmod']['chat'] = false;
 $wgGroupPermissions['autopatrol']['autopatrol'] = true;
 $wgGroupPermissions['interwiki-admin']['interwiki'] = true;
-$wgGroupPermissions['template-editor']['template-editor'] = true;
-$wgGroupPermissions['sysop']['template-editor'] = true;
 $wgAddGroups['sysop'] = array( 'editor', 'reviewer', 'autoreview', 'autopatrol', 'patroller', 'translateadmin', 'confirmed', 'researcher', 'chatmod', 'blockedfromchat', 'template-editor' );
```
Comment (MacFan4000) [inline LocalSettings.php]: Let’s also remove template-editor from this array (line 361) as well as on line 362 please.

**Review by MacFan4000 — CHANGES_REQUESTED** [review]

File: LocalSettings.php (hunk 1 above)
Comment (dmehus) [reply LocalSettings.php]: @MacFan4000 Are lines 361-362 the `$wgAddGroups` lines? Assuming that's the case, I was just thinking that. 👍

Comment (dmehus) [reply LocalSettings.php]: @Justarandomamerican, have you also removed `template-editor` permissions from any users that had it assigned on-wiki? Not sure if required, but I think it's a good practice.

Comment (Justarandomamerican) [reply LocalSettings.php]: @dmehus I'll do that later.

**Comment by Justarandomamerican** [comment]
Accidentally closed.

File: LocalSettings.php (hunk 1 above)
Comment (MacFan4000) [reply LocalSettings.php]: That will need to be done before I merge this, otherwise it won’t be possible.

Comment (Justarandomamerican) [reply LocalSettings.php]: **Done**

**Review by MacFan4000 — APPROVED** [review]

**Pull request merged**